### PR TITLE
ACCESSFND-1493 - OIDC Feature in Jenkins

### PR DESCRIPTION
**Pull request by acarlson0000**

### CODE REVIEW

File: configure_envoy/src/test_configure.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+            ]
+        }
+    }
+
+    updated_configuration = insert_filter(blank_configuration)
+    config_string = updated_configuration["static_resources"]["listeners"][0]["filter_chains"][0]["filters"][0]["typed_config"]["http_filters"][0]["typed_config"]["config"]["configuration"]["value"]
+    exclude_paths = json.loads(config_string)['exclude_paths']
+    assert exclude_paths == ['/favicon.ico']
+
+def test_anonymous_paths_when_set(monkeypatch):
```
Comment (robcatt) [inline configure_envoy/src/test_configure.py:24]: Where does monkeypatch come from? Is it provided by the unittest test runner?

**Review by robcatt — APPROVED** [review]
Looks good, just got a question on the testing.

File: configure_envoy/src/test_configure.py (hunk 1 above)
Comment (acarlson0000) [reply configure_envoy/src/test_configure.py:24]: one for @ahmed-adam

Comment (ahmed-adam) [reply configure_envoy/src/test_configure.py:24]: It's from pytest: https://docs.pytest.org/en/4.6.x/monkeypatch.html

Comment (acarlson0000) [reply configure_envoy/src/test_configure.py:24]: @robcatt  - https://docs.pytest.org/en/4.6.x/monkeypatch.html

**Pull request merged**